### ISSUE TITLE
Example of retrieve an option options-pages-with-submenus.php

### ISSUE THREAD

**Issue by serggiocaramello** [CMB2/CMB2#1484]
Hello,

I have been trying get the data of the first tab in this [example ](https://github.com/CMB2/CMB2-Snippet-Library/blob/master/options-and-settings-pages/options-pages-with-tabs-and-submenus.php) but i cant make it work.

Could you please show me the correct way for access the first tab data.

Thanks.

**Comment by tw2113** [comment]
Yep yep. The `object-type` parameter is what determines everything. In this case it's an option page. other options would be the post types, which would be post meta, 'term', i think, would be term meta, and 'user' would be user meta.
